pkg/profiler/cpu/metrics.go

```diff
@@ -60,5 +61,13 @@ func newMetrics(reg prometheus.Registerer) *metrics {
 				NativeHistogramBucketFactor: 1.1,
 			},
 		),
+		obtainMapMetrics: promauto.With(reg).NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name:        "parca_agent_bpf_map_memlock",
+				Help:        "Metrics for BPF maps created by the agent",
+				ConstLabels: map[string]string{"type": "cpu"},
+			},
+			[]string{"bpf_map_name", "bpf_map_keySize", "bpf_map_valueSize", "bpf_max_entries"},
```

Comment:
This doesn't look quite right, I would expect to have a distinct metric for entries, key size, value size and memlock value. Only `bpf_map_name` sounds like something that should be a label.